components/src/core/ChannelValue/ChannelValue.tsx

```diff
@@ -1,40 +1,16 @@
 import React, { useCallback } from 'react';
 import Typography from '@material-ui/core/Typography';
-import { makeStyles, Theme, useTheme } from '@material-ui/core/styles';
+import { makeStyles, Theme, createStyles } from '@material-ui/core/styles';
 import clsx from 'clsx';
 import PropTypes from 'prop-types';
 
-const useStyles = makeStyles((theme: Theme) => ({
-    root: {
-        display: 'inline-flex',
-        alignItems: 'center',
-        lineHeight: 1.2,
-    },
-    icon: {
-        marginRight: theme.spacing(0.5),
-        display: 'inline',
-        fontSize: 'inherit',
-    },
-    text: {
-        fontSize: 'inherit',
-        lineHeight: 'inherit',
-        letterSpacing: 0,
-    },
-    units: {
-        fontWeight: 300,
-    },
-    value: {
-        fontWeight: 600,
-    },
-}));
-
 export type ChannelValueClasses = {
     root?: string;
     icon?: string;
     units?: string;
     value?: string;
 };
-export type ChannelValueProps = {
+export type ChannelValueProps = Omit<React.HTMLAttributes<HTMLSpanElement>, 'prefix'> & {
```

Comment:
can we 
```
import React, {HTMLAttributes} from 'react'
```
above to make this a little easier to read?

And do we need to omit prefix here if our prefix property will overwrite it?

Reply:
`Proptypes` doesn't like it for some reason if I don't do the omit.

Reply:
Should I replace all the `React.FC` to `FC` with imports too?

Reply:
I think what you have is fine, React.FC doesn't bother me